blog/from-acropolis-to-rome/README.md

```diff
@@ -0,0 +1,71 @@
+# Post
+
+#### Title
+
+From Acropolis To Rome
+
+#### Purpose
+
+Let our community know about the transition between Acropolis and Rome.
+
+#### url
+
+blog.joystream.org/from-acropolis-to-rome/
+
+#### Cover
+
+<p align="center"><img src="from-acropolis-to-rome.png"></p>
+
+#### Lead
+
+We bid farewell to Acropolis to welcome our new testnet, Rome!
+
+#### Body
+
+After almost nine months of experimentation by testnet participants on Acropolis, we are now ready to move on to our latest release, codenamed Rome, which incorporates a variety of new features.
+
+The Plan
+--------
+
+The main aspect of the release process for participants to be aware of is our intention to "kill" the Acropolis chain this Friday (13th March). The snapshot for Acropolis was taken at block number `4,718,300`.
```

Comment:
- Consider merging these three sections to two.
- Clarify that `Validators` and `Storage Providers` will receive a full weeks payment, but will not start with any privileges for `Rome.